src/data/customer/Order.js

```diff
@@ -802,3 +831,55 @@ export const deleteOrderDispatchAsync = async (orderId, token) => {
     return apiResponseErrorHandler(e);
   }
 };
+
+/**
+ * Retrieve Order Status reason
+ * @param {int} statusId
+ * @param {string} token
+ * @return {object} Promise resolve/reject
+ //TODO: needs more extensive unit testing
+ */
+export const getOrderStatusReasonAsync = async (statusId, token) => {
+  try {
+    let response = await axios({
+      method: 'GET',
+      url: `${endpoints.API_V3.ORDER_STATUS_REASON}`,
```

Comment:
@paul-carpal where is this being used?

Reply:
@maartenhemmes this is used for retrieving reasons for cancellation.

<img width="1365" alt="Screenshot 2019-10-03 at 9 52 29 AM" src="https://user-images.githubusercontent.com/6081485/66093600-88527700-e5c3-11e9-9e2f-4366029e582f.png">